ui/apps/platform/src/Containers/Collections/CollectionsPage.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import NotFoundMessage from 'Components/NotFoundMessage';
+import CollectionsTablePage from './CollectionsTablePage';
+
+function CollectionsPage() {
+    // TODO Implement permissions once https://issues.redhat.com/browse/ROX-12619 is merged
+    // const { hasReadAccess, hasReadWriteAccess } = usePermissions();
+    const hasReadAccessForCollections = true; // hasReadAccess('TODO');
+    const hasWriteAccessForCollections = true; // hasReadWriteAccess('TODO');
+
+    if (!hasReadAccessForCollections) {
+        return <NotFoundMessage title="404: We couldn't find that page" />;
+    }
```

Comment:
Dave, can you lift role permission up to Body.tsx and NavigationSidebar.tsx files?

That is, conditional rendering depends on feature flag enabled **and** access to resource.

When we circle back to #2105 soon, both conditional rendering requirements will become declarative instead of imperative, therefore minimal code changes to add a new route.